rbac/providers/ldap/initial_inbound_sync.py

```diff
@@ -122,6 +118,8 @@ def initialize_ldap_sync():
     if LDAP_DC:
         connect_to_db()
```

Comment:
Unrelated to this PR, but still in use here: connect_to_db()

Reminder: You will need to be returning a connection from that and using it and/or connecting on a per-query basis since RethinkDb is not thread safe. Absolutely do not open a REPL in the background and rely on it for database operations.

Reply:
We already have a task open to fix that, #580